src/mumble/VolumeSliderWidgetAction.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "VolumeSliderWidgetAction.h"
+#include "VolumeAdjustment.h"
+
+#include <cmath>
```

Comment:
I've seen this include in multiple files in your PR, but I don't think you require it once - probably a remnant from the old implementation?